### PR TITLE
Oreos

### PR DESCRIPTION
**Pull request by teancom**
Now against the right branch!

### CODE REVIEW

File: _posts/2020-11-30-homemade-oreos.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "Homemade Oreos"
+date:   2020-11-29
```
Comment (aloria) [inline _posts/2020-11-30-homemade-oreos.markdown:4]: Can you change this to 11/30?

Comment (teancom) [reply _posts/2020-11-30-homemade-oreos.markdown]: Wait, you expect me to *read* and *think* about what I'm copying??? How rude.

(fixed)

**Comment by aloria** [comment]
🎉

File: _posts/2020-11-30-homemade-oreos.markdown (hunk 1 above)
Comment (aloria) [reply _posts/2020-11-30-homemade-oreos.markdown]: I would like a recipe for copypasta on my desk by end of week 😛

Comment (teancom) [reply _posts/2020-11-30-homemade-oreos.markdown]: You failed to reckon with the fact that I've been on vacation for a week and don't want to actually get back to work. 😄  https://github.com/aloria/aloria.github.io/pull/6

**Pull request merged**